ext/ldap/config.m4

```diff
@@ -59,52 +60,53 @@ if test "$PHP_LDAP" != "no"; then
     [-DZEND_ENABLE_STATIC_TSRMLS_CACHE=1])
 
   AS_VAR_IF([PHP_LDAP], [yes], [
-    for i in /usr/local /usr; do
-      PHP_LDAP_CHECKS([$i])
-    done
+    PKG_CHECK_MODULES([LDAP], [lber ldap])
+    PHP_LDAP_PKGCONFIG=true
   ], [PHP_LDAP_CHECKS([$PHP_LDAP])])
 
-  AS_VAR_IF([LDAP_DIR],, [AC_MSG_ERROR([Cannot find ldap.h])])
+  if test "$PHP_LDAP_PKGCONFIG" = true; then
```

Comment:
Please use `AS_IF`, especially whenever inside another macro. Newer autoconf docs have changed to unconditionally recommending it. Not using `AS_IF` can blow up at odd times in hard-to-debug ways.